docs/add-ins/addin-tutorial.md

```diff
@@ -141,80 +253,63 @@ Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this li
+
+    ```xml
+    <bt:String id="groupLabel" DefaultValue="Git the gist"/>
+    ```
 
-   ```xml
-    <bt:String id="groupLabel" DefaultValue="Git the Gist"/>
-   ```
 1. Add the following elements as children of the `bt:ShortStrings` element.
 
-   ```xml
-    <bt:String id="insertGistLabel" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertGistTitle" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistLabel" DefaultValue="Insert gist"/>
+    <bt:String id="insertGistTitle" DefaultValue="Insert gist"/>
+    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert default gist"/>
+    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert default gist"/>
+    ```
+
 1. Add the following elements as children of the `bt:LongStrings` element.
 
-   ```xml
-    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your Gists and allows you to insert their contents into the current message">
-      <bt:Override Locale="es-ES" Value="Muestra una lista de sus Gists y permite insertar su contenido en el mensaje actual"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the Gist you mark as default into the current message">
-      <bt:Override Locale="es-ES" Value="Inserta el contenido de lo Gist que marca como predeterminado en el mensaje actual"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your gists and allows you to insert their contents into the current message"/>
+    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the gist you mark as default into the current message"/>
+    ```
 
-This defines the string values that will be used for the add-in. It also localizes the strings into Spanish by providing a `bt:Override` element for each string. Additional languages can be added as additional `bt:Override` elements. The `DefaultValue` is used for clients that use the locale specified in the `DefaultLocale` element in the manifest.
+1. Save your changes to the manifest. 
 
-Save your changes to the manifest. Since we installed the add-in from a file, we need to reinstall it in order for the changes to take effect.
 
-1. Open Outlook 2016. On the **Home** tab in the ribbon, click the **Store** button.
+### Reinstall the add-in
```

Comment:
Interesting suggestion. I opted for an H3 heading here because:
- This task is considered part of the process for adding new buttons -- i.e., before you'll see those new buttons in Outlook, you need to reinstall the add-in. (So this section should appear within the H2 "Defining buttons" section). 
- In a tutorial article, it's good practice for the 5 tasks listed in the checklist at the top of the tutorial to be the only H2s within the main part of the article (those are the links rendered on the right side of the page, and allow the reader to see at a glance the core tasks that the tutorial covers).